### ISSUE TITLE
Competitor wants to merge into Ansible

### ISSUE THREAD

**Issue by bbaassssiiee** [evandam/evandam.conda#12]
There is another library that competes in this space. A PR into Ansible is filed, but stuck in limbo...

- https://github.com/UDST/ansible-conda/issues/1
- https://github.com/ansible/ansible/pull/40455

**Comment by evandam** [comment]
I used UDST/ansible-conda for a while before creating my own...this realistically should have been a fork of that project.

I'd love to see this get merged into ansible core, but there's definitely some added time and effort that needs to go into making that happen that I'm just not able to take on right now. We also lose a lot of the flexibility with pushing new releases, so I'd want to make sure this is rock solid before starting that process.

**Comment by bbaassssiiee** [comment]
Closing this because ansible 2.10 has collections.
